blazor/treegrid/rows/detail-template.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Detail Template in Blazor Tree Grid Component | Syncfusion
+description: Checkout and learn here all about Detail Template in Syncfusion Blazor Tree Grid component and much more details.

```

Comment:
description: Check out here and learn more details about the Detail Template in the Syncfusion Blazor Tree Grid component.